src/test/java/core/basesyntax/service/impl/ProcessServiceImplTest.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.model.FruitTransaction;
+import java.util.List;
+import org.junit.Test;
+
+public class ProcessServiceImplTest {
+    private ProcessServiceImpl processService = new ProcessServiceImpl();
+
+    @Test
+    public void process_return_Ok() {
+        List<String> actual = List.of("type,fruit,quantity",
+                "b,orange,44",
+                "b,banana,20",
+                "s,orange,75");
+        List<FruitTransaction> expected = List.of(
+                new FruitTransaction(FruitTransaction.Operation.BALANCE, "orange", 44),
+                new FruitTransaction(FruitTransaction.Operation.BALANCE, "banana", 20),
+                new FruitTransaction(FruitTransaction.Operation.SUPPLY, "orange", 75));
+        assertEquals(expected, processService.getTransactions(actual));
+    }
+
```

Comment:
empty line
Maybe we could test for blank input, for wrong operation letter like "y,banana,50" , for missing data in line like "b,,50"

Reply:
good point, you should cover all possible and impossible cases you can find